help/destinations/catalog/crm/zendesk.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Zendesk connection
+description: The Zendesk destination allows you to export your account data and activate it within Zendesk for your business needs.
+---
+# [!DNL Zendesk] connection
+
+[!DNL Zendesk](https://www.zendesk.com) is a customer service solution and sales tool.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Zendesk] Contacts API](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/), to create and update identities within a segment as contacts within [!DNL Zendesk].
+
+[!DNL Zendesk] uses bearer tokens as an authentication mechanism to communicate with the [!DNL Zendesk] Contacts API. Instructions to authenticate to your [!DNL Zendesk] instance are further below, in the [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can deliver personalized experiences to your users, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Zendesk], to display in the users' feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Zendesk] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Experience Platform documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
+
+### [!DNL Zendesk] prerequisites {#prerequisites-destination}
+
+In order to export data from Platform to your [!DNL Zendesk] account you need to have a [!DNL Zendesk] account.
+
+#### Gather [!DNL Zendesk] credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Zendesk] destination:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| `Bearer Token` | The Access Token you have generated in your [!DNL Zendesk] account. <br> Follow the documentation to [generate a [!DNL Zendesk] access token](https://developer.zendesk.com/documentation/sales-crm/first-call/#1-generate-an-access-token) if you do not have one.| `a0b1c2d3e4...v20w21x22y23z` |
```

Comment:
"Bearer token" and "access token". No need to capitalize, as it's a common name.